### PR TITLE
chore: docs site

### PR DESCRIPTION
**Pull request by vladfrangu**
**Please describe the changes this PR makes and why it should be merged:**

We get a website for types now. 🎉 

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

### CODE REVIEW

**Comment by vladfrangu** [comment]
discord-api-types
discord-api-types-long-rainbow

File: website/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+	"editor.detectIndentation": true,
+	"files.eol": "\n",
+	"files.associations": {
+		"*.mdx": "markdown"
+	},
+	"cSpell.language": "en-GB",
```
Comment (kyranet) [inline website/.vscode/settings.json:7]: discordjs uses en-US.

File: website/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# The MIT License (MIT)
+
+Copyright © `2020` `The Sapphire Community and its contributors`
```
Comment (kyranet) [inline website/LICENSE.md:3]: Is it intended that you don't plan on sublicensing?

File: website/src/components/Home/HomepageFeatures.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```
Comment (kyranet) [inline website/src/components/Home/HomepageFeatures.tsx:5]: Is it? The only LICENSE I find is Sapphire's.

File: website/src/components/Home/InstallButton/InstallTabButton.module.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.button {
+	box-shadow: var(--sapphire-paper-drop-shadow-4);
```
Comment (kyranet) [inline website/src/components/Home/InstallButton/InstallTabButton.module.css:2]: I suppose it's intentional you use Sapphire's variables?

File: website/src/utils/Intl.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const IntlFormatter = new Intl.NumberFormat('en-GB', { maximumFractionDigits: 2 });
```
Comment (kyranet) [inline website/src/utils/Intl.ts:1]: Why should it be exclusively in en-GB? Why not en-US or let the browser decide based on the user's locale?

File: website/.vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+	"search.exclude": {
+		"**/.docusaurus/": true,
+		"**/.git/": true,
+		"**/.yarn": true,
+		"**/*.code-search": true,
+		"**/bower_components": true,
+		"**/node_modules": true,
+		"**/projects/": true
+	},
+	"cSpell.words": ["favna", "nsfw"]
```
[the remark this answers is not in the document]
Comment (vladfrangu) [reply website/.vscode/settings.json:19]: Considering its literally a copy of the sapphire website with some tweaks, this is why a lot of files reference our tooling god Favna and Sapphire 😁

File: website/src/utils/Intl.ts (hunk 5 above)
Comment (vladfrangu) [reply website/src/utils/Intl.ts:1]: How can I do that? Although ideally it should stick to en-US or en-GB or even better, the locale of the website if we'll add i18n

Comment (kyranet) [reply website/src/utils/Intl.ts:1]: You use `undefined` for automatic detection, but honestly, you can also use `en-US`, so the formatting is consistent with the rest of the website.

File: website/LICENSE.md (hunk 2 above)
Comment (vladfrangu) [reply website/LICENSE.md:3]: Yeah, doesn't really matter

**Review by kyranet — APPROVED** [review]

**Pull request merged**